Mathlib/RingTheory/Unramified/Basic.lean

```diff
@@ -214,4 +225,51 @@ theorem FormallyUnramified.localization_map [FormallyUnramified R S] :
 
 end Localization
 
+end FormallyUnramified
+
+section
+
+variable (R : Type u) [CommSemiring R]
+variable (A : Type u) [Semiring A] [Algebra R A]
+
+/-- An `R`-algebra `A` is unramified if it is formally unramified and of finite type. -/
```

Comment:
Given that there is more than one definition in the literature, I think that the docstring should definitely mention this and perhaps explain that we are using the stacks definition rather than the EGA definition (there is a correct way to format references so that they work in the API docs BTW). Other than that, this looks ready to go to me.

Reply:
We added a remark to the docstring. Maybe we should make a separate PR adding all the common alg.geo references to the bibliography.